io.github.MakovWait.Godots.json

```diff
@@ -0,0 +1,108 @@
+{
+    "id" : "io.github.MakovWait.Godots",
+    "runtime" : "org.freedesktop.Sdk",
+    "runtime-version" : "23.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11",
+        "org.freedesktop.Sdk.Extension.dotnet7"
+    ],
+    "command": "godots",
+    "build-options": {
+        "append-path": "/usr/lib/sdk/dotnet7/bin",
+        "append-ld-library-path": "/usr/lib/sdk/dotnet7/lib",
+        "env": {
+            "PKG_CONFIG_PATH": "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig"
+        }
+    },
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=x11",
+        "--device=all",
+        "--socket=pulseaudio",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Flatpak",
```

Comment:
this DBus access is not allowed by default. Why do you need it?

Reply:
`--take-name=org.freedesktop.Flatpak`? It's for allowing Godot to launch code editors like Visual Studio Code and for the Blender importer. The existing Godot flatpak has the same permission.

Reply:
two wrongs don't make a right.

Reply:
```
        "finish-args-flatpak-spawn-access"
```
that's the build failure.

Reply:
Would it be okay to mention overriding this permission in the appstream description for supporting external editors and Blender? For many users, this would be a requirement for C# support or importing blend files. I could also see if I could submit a PR upstream to popup a message if the permission isn't granted.

Reply:
did you read that part of the documentation?

https://docs.flathub.org/docs/for-app-authors/linter/

Reply:
Must have missed that. I'll submit a PR :+1: